### DISCUSSION TITLE
¿Cómo convertir un texto a multilínea en la tabla de atributos añadido a la composición de mapa?

### DISCUSSION THREAD

**Discussion by Fertuesta25** [qgispe/discusiones#9] [Preguntas&Respuestas]
Esta pregunta se ha generado a raíz de la consulta realizada por William Hidalgo en el grupo de telegram de QGIS Perú.

"Amigos buenas noches, como hago para modificar esta etiqueta muy larga"

![photo5044240597694654689](https://user-images.githubusercontent.com/59838508/121581374-5e399500-c9f3-11eb-808e-535b57572225.jpg)

**Comment by Fertuesta25** [comment]
En respuesta a esta pregunta he sugerido la siguiente solución:

Se debe hacer uso de la función `wordwrape`, esta función te pide una cadena de texto y longitud como parámetros obligatorios. La cadena de texto pueden ser los valores almacenados en un campo, y la longitud se tendría que ir probando hasta que el texto se acomode a nuestras necesidades.

**Reply by ambarja** [reply]
Totalmente de acuerdo con @Fertuesta25, aquí adjunto algunas imágenes de referencia 
![Captura de pantalla de 2021-06-10 14-46-43](https://user-images.githubusercontent.com/23284899/121587725-baec7e00-c9fa-11eb-9430-2b2004039672.png)
<br>
![Captura de pantalla de 2021-06-10 14-42-40](https://user-images.githubusercontent.com/23284899/121587796-ccce2100-c9fa-11eb-8e83-0138ef853392.png)

**Reply by Fertuesta25** [reply]
Gracias Antony por las imágenes de referencia.
